java/dev/checkstyle/checkstyle.xml

```diff
@@ -195,10 +195,6 @@
             <property name="lineWrappingIndentation" value="4"/>
             <property name="arrayInitIndent" value="2"/>
         </module>
-        <module name="AbbreviationAsWordInName">
```

Comment:
This conflicts with some of the variable names we use as constants and it made the most sense to just remove this rule